spec/factories/installations.rb

```diff
@@ -1,9 +1,9 @@
 FactoryBot.define do
   factory :installation do
     sequence(:name) { |n| "Installation #{n}" }
-    latitude 38.6921631
-    # rubocop:disable Lint/AmbiguousOperator
-    longitude -77.1374000999999
-    # rubocop:enable Lint/AmbiguousOperator
+
+    after(:create) do |installation|
+      installation.location = create(:location, locatable: installation)
+    end
```

Comment:
This was the most reliable way to create an associated location in a factory. The FactoryBot documentation is a bit hard-to-follow regarding associations, but this seems to do the trick (here and in other "locatable" models)…